### PR TITLE
add force_send_invite variable to user module

### PR DESCRIPTION
**Pull request by davidbloss**
Add `force_send_invite` variable to user module.

Introduced in [this TF PR](https://github.com/OpsLevel/terraform-provider-opslevel/pull/475)

### CODE REVIEW

File: modules/user/variables.tf [hunk 1]
```diff
@@ -3,6 +3,12 @@ variable "email" {
   description = "The email address of the user."
 }
 
+variable "force_send_invite" {
```
Comment (rocktavious) [inline modules/user/variables.tf:6]:
```suggestion
variable "send_invite" {
```

File: modules/user/variables.tf [hunk 2]
```diff
@@ -3,6 +3,12 @@ variable "email" {
...
+  type        = bool
+  description = "Send an invite email even if notifications are disabled for the account."
```
Comment (rocktavious) [inline modules/user/variables.tf:8]:
```suggestion
  description = "Choose weather to send an invite email even if notifications are disabled for the account."
```

File: modules/user/variables.tf [hunk 3]
```diff
@@ -3,6 +3,12 @@ variable "email" {
...
+  default     = null
```
Comment (rocktavious) [inline modules/user/variables.tf:9]:
```suggestion
  default     = true
```

File: modules/user/main.tf [hunk 4]
```diff
@@ -1,7 +1,8 @@
 data "opslevel_users" "all" {}
 
 resource "opslevel_user" "this" {
-  email              = var.email
+  email = var.email
+  # force_send_invite  = var.force_send_invite  # TODO: uncomment on next release
```
Comment (rocktavious) [inline modules/user/main.tf:5]:
```suggestion
  email                       = var.email
  force_send_invite  = var.force_send_invite
```

We can stage this but just not merge it till the TF provider is released

**Comment by davidbloss** [comment]
Replaced by [PR #25](https://github.com/OpsLevel/terraform-opslevel-modules/pull/25)

**Pull request closed without merging**